src/terminfo/parm.rs

```diff
@@ -18,6 +18,7 @@ use std::iter::repeat;
 #[derive(Clone, Copy, PartialEq)]
 enum States {
     Nothing,
+    Ignore,
```

Comment:
We should probably explicitly call this `Delay`. If we decide to ignore something else, we'll need to introduce an additional state anyways.